op-node/flags/p2p_flags.go

```diff
@@ -53,6 +53,7 @@ var (
 	GossipMeshDlazyName    = "p2p.gossip.mesh.dlazy"
 	GossipFloodPublishName = "p2p.gossip.mesh.floodpublish"
 	SyncReqRespName        = "p2p.sync.req-resp"
+	P2PPingingName         = "p2p.pinging"
```

Comment:
Not sure about the name "pinging", specifically the "ing" aspect. Looking at the lines above I would expect this to be `p2p.gossip.mesh.ping`, as these pings are amongst gossip nodes.

Reply:
Gossip is only a subset of the p2p stack, and its own isolated sub-module in the p2p stack. This ping-protocol does not use gossipsub and has nothing to do with gossipsub meshing as it pings all connected peers, nothing specific to topics. I'll rename it to `p2p.ping` if the `ing` part is not the right style.